backend/api.test/Services/RobotService.cs

```diff
@@ -56,15 +59,23 @@ public async Task Read()
         [Fact]
         public async Task ReadIdDoesNotExist()
         {
-            var robotService = new RobotService(_context, _logger, _robotModelService, _signalRService);
+            var robotService = new RobotService(_context, _logger, _robotModelService, _signalRService, _accessRoleService);
             var robot = await robotService.ReadById("some_id_that_does_not_exist");
             Assert.Null(robot);
         }
 
         [Fact]
         public async Task Create()
         {
-            var robotService = new RobotService(_context, _logger, _robotModelService, _signalRService);
+            var robotService = new RobotService(_context, _logger, _robotModelService, _signalRService, _accessRoleService);
+            var installationService = new InstallationService(_context, _accessRoleService);
+
+            var installation = await installationService.Create(new CreateInstallationQuery
+            {
+                Name = "Johan Sverdrup",
```

Comment:
Huldra as test asset

Reply:
Then we will need to use Huldra in initDb as well I think, which may also change Dev, but that is probably fine